easybuild/easyconfigs/s/SPAdes/SPAdes-3.6.2-goolf-1.4.10.eb

```diff
@@ -0,0 +1,35 @@
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+# Author: Pablo Escobar Lopez
+# sciCORE - University of Basel
+# SIB Swiss Institute of Bioinformatics 
+
+easyblock = "CMakeMake"
+
+name = 'SPAdes'
+version = '3.6.2'
+
+homepage = 'http://bioinf.spbau.ru/en/spades'
+description = """Genome assembler for single-cell and isolates data sets"""
+
+toolchain = {'name': 'goolf', 'version': '1.4.10'}
+
+source_urls = ['http://spades.bioinf.spbau.ru/release%(version)s/']
+sources = [SOURCE_TAR_GZ]
+
+builddependencies = [('CMake', '2.8.12')]
+
+dependencies = [('zlib', '1.2.8')]
```

Comment:
is bzip a missing dep here too?

cfr. https://lists.ugent.be/wws/arc/easybuild/2016-04/msg00055.html